galaxyui/src/app/home/home.routing.module.ts

```diff
@@ -8,7 +8,7 @@ import { ContentBlockResolver, VendorListResolver } from './home.resolver.servic
 
 const homeRoutes: Routes = [
     {
-        path: 'home',
+        path: '',
```

Comment:
removing `home` will create problems with other links in the app. I recommend doing something like this:

```
const homeRoutes: Routes = [
    {
        path: '',
        component: HomeComponent,
        resolve: {
            vendors: VendorListResolver,
            contentBlocks: ContentBlockResolver,
        },
    },

    {
        path: 'home',
        component: HomeComponent,
        resolve: {
            vendors: VendorListResolver,
            contentBlocks: ContentBlockResolver,
        },
    },
];
```